### ISSUE TITLE
Сделать переиспользуемые заголовки для всей страницы

### ISSUE THREAD

**Issue by Anastacia-Tesli** [cohorts51-52/lubimovka#35]
Я думаю нужны заголовки секций с одинаковыми классами и стилями, чтобы не изобретать велосипед )

**Comment by Yauheni5** [comment]
Да, на начальном этапе это упустили. Может когда финишные правки будем вносить переделаем?

**Comment by Anastacia-Tesli** [comment]
Можно и переделать, но как-то не очень это эффективно

**Comment by Anastacia-Tesli** [comment]
@Yauheni5 а ты в своих разных блоках сейчас как делаешь? Не переиспользуешь готовый заголовок?

**Comment by Yauheni5** [comment]
я не переиспользовал, но могу исправить это

**Comment by Yauheni5** [comment]
хотя мне кажется лучше это исправить на готовом варианте сразу у всех. Могу на себя повесить данную задачу
